sdk/websites/Azure.ResourceManager.AppService/samples/Generated/Samples/Sample_SubscriptionResourceExtensions.cs

```diff
@@ -539,7 +539,7 @@ public async Task VerifyHostingEnvironmentVnet_VerifyHostingEnvironmentVnet()
         public async Task GetGetUsagesInLocations_GetUsagesInLocationForSubscription()
         {
             // Generated from example definition: specification/web/resource-manager/Microsoft.Web/stable/2024-04-01/examples/GetUsagesInLocation.json
-            // this example is just showing the usage of "GetUsagesInLocation_List" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "GetUsagesInLocation_list" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `list`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/ResourceProvider.json#L551